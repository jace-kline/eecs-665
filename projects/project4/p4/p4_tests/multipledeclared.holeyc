int a;
bool b;

int main(int argc, charptr argv) {
    int a;
    a = 5;
    boolptr a;
    b = false;
    a = ^b;
    bool a;
    a = b;
    return 0;
}