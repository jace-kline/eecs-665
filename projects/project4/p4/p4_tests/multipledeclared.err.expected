FATAL [7,13]: Multiply declared identifier
FATAL [10,10]: Multiply declared identifier
Name Analysis Failed
